fmf/context.py

```diff
@@ -44,6 +44,9 @@ def __init__(self, origin):
         else:
             self._to_compare = self._split_to_version(origin)
 
+        # Store the original string for regexp processing
+        self.origin = origin
```

Comment:
Wouldn't `raw` be a better name for this variable? I find it more readable, especially when it is used in operators. Until now 'origin' was local to this init method so it didn't matter what was its name. (and yes, I'm bad at naming things :)

Reply:
Good idea, renamed in bd1ec88.